packages/http-client/src/ceramic-http-client.ts

```diff
@@ -78,7 +78,7 @@ export class CeramicClient implements CeramicApi {
   constructor(apiHost: string = CERAMIC_HOST, config: Partial<CeramicClientConfig> = {}) {
     this._config = { ...DEFAULT_CLIENT_CONFIG, ...config }
 
-    this._apiUrl = new URL(API_PATH, apiHost)
+    this._apiUrl = new URL(apiHost.replace(/\/$/, '') + API_PATH) 
```

Comment:
I personally find regexes pretty inscrutable.  Can you maybe add a constant for this regex with a comment on what it does?

Reply:
This also feels like the kind of thing that the URL class should be able to handle and abstract away for us without us needing to do raw string manipulation.  For one thing I'd be worried about how cross-platform the string-manipulation-based approach will be - have you tested this on windows?

Reply:
`apiHost` could be provided with a slash or without a slash, or maybe with multiple slashes at the end (highly unlikely). The patch handles the first two cases by effectively replacing the single slash. Then `API_PATH` provides a slash symbol. That works.

A single thing really missing here to my taste is a unit test that goes through scenarios of zero, one slashes.

Reply:
Just circling back to this - thank you @stbrody and @ukstv for your feedback! I'll start by adding in the unit tests. As for the regex - are we fine with this?

Reply:
@ukstv  - went in a slightly different direction compared to the original patch. Let me know if this looks good to you!